gulpfile.babel.js

```diff
@@ -39,14 +43,20 @@ gulp.task('babel', () =>
     .pipe(gulp.dest('dist'))
 );
 
+gulp.task('build:frontend', () =>
```

Comment:
Will you be eventually considering removing gulp from the project and passing all tasks to webpack?